<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixInputDialog</class>
 <widget class="QDialog" name="MatrixInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabUser">
      <attribute name="title">
       <string>User</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Representation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="representationComboBoxUser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textEditUser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRaw">
      <attribute name="title">
       <string>Raw</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Representation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="representationComboBoxRaw">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textEditRaw"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MatrixInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MatrixInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>representationComboBoxRaw</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MatrixInputDialog</receiver>
   <slot>rawRepresentationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>representationComboBoxUser</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MatrixInputDialog</receiver>
   <slot>userRepresentationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditRaw</sender>
   <signal>textChanged()</signal>
   <receiver>MatrixInputDialog</receiver>
   <slot>rawTextEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditUser</sender>
   <signal>textChanged()</signal>
   <receiver>MatrixInputDialog</receiver>
   <slot>userTextEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>userRepresentationChanged(int)</slot>
  <slot>rawRepresentationChanged(int)</slot>
  <slot>rawTextEditChanged()</slot>
  <slot>userTextEditChanged()</slot>
 </slots>
</ui>
